apps/iam/kinds/v0alpha1/serviceaccountspec.cue

```diff
@@ -1,6 +1,12 @@
 package v0alpha1
 
 ServiceAccountSpec: {
+    avatarUrl: string
+    disabled: bool |* false
+    external: bool |* false
+    login: string
```

Comment:
Which of these fields are actually used?  avatar and login do not make sense for service accounts -- they technically exist now because we share the same SQL table as regular users.

external?  Is that possible with our current setup?

NOTE we can always *ADD* fields later, but we can not remove fields without bumping the apiVersion 😜